shared/js/media/video_player.js

```diff
@@ -491,6 +502,25 @@ function VideoPlayer(container) {
       }
     });
   }
+
+  function localize() {
+    // XXX: Ideally, we would add the duration too, but that is not
+    // available via fileinfo metadata yet.
+    var label;
+    var orientation = rotation === 90 || rotation === 270 ?
+      'orientationPortrait' : 'orientationLandscape';
+    if (videotimestamp) {
+      var locale_entry = navigator.mozL10n.get('videoDescription',
+        { orientation: navigator.mozL10n.get(orientation) });
+      label = MediaUtils.formatDate(videotimestamp, locale_entry);
+    } else {
+      label = navigator.mozL10n.get('videoDescriptionNoTimestamp',
+        { orientation: navigator.mozL10n.get(orientation) });
+    }
+    poster.setAttribute('aria-label', label);
+  }
```

Comment:
Like I said above, I tested it and all l10n functions fail gracefully by returning empty strings.